_data/gallery.yaml

```diff
@@ -141,6 +141,6 @@
 - title: Radiation-Map
   description: Radiation map produced with data from the nuclear safety institute
   url: voila/render/Radiation.ipynb
-  ref: 6a665e3648fe81436cf0af8b23c439e204042a85
+  ref: c1f8ae098a0b1ad1390bcd75d4419508a7bd0219
   repo_url: https://github.com/MackyDIARRA/radiation.git
```

Comment:
You might need to remove the `.git` from there?

Reply:
isn't it the same thing without ".git"?

Reply:
I'm not sure how this `repo_url` parameter is used. It might work. But given other examples don't have it it might be safer to remove it.